EmployeeManagementService/EmployeeManagementService.Infrastructure/Persistence/EmployeeRepository.cs

```diff
@@ -9,12 +9,14 @@ namespace EmployeeManagementService.Infrastructure.Persistence
 {
     public interface IEmployeeRepository
     {
-        Task CreateEmployee(string firstName, string lastName, string username, string ssn, byte[] password, string role, bool? active = true);
+        Task CreateEmployee(string firstName, string lastName, string username, string ssn, byte[] password, string role, string a1, string a2, string city, string state, string zip, bool? active = true);

```

Comment:
We may need to change the way we do this in refactoring. Before we call create in service, we map domain to entity first and just pass in the entity.